### DISCUSSION TITLE
Raid Fleet Delay

### DISCUSSION THREAD

**Discussion by LepRyot** [endless-sky/endless-sky#9705] [Balance, Mechanics, and Interactions]
Hi, do you think that it could be more realistic if that pirate raids don't immediately become aware of the player's cargo to dps makeup? Information can't travel instantly across space. I know there's a reason for the pirate raid mechanic and I don't want to undermine freighter balancing. My concern is that the consequences kick in just a wee bit too early

Imagine I have a fully defended fleet one day but then decide to buy 10 bulk freighters while at the next shipyard. The moment I depart, pirates from kingdom come are waiting in orbit wanting a piece of me. The bottom line is how did they know? If they weren't already in the system before I bought the freighters then how are they here now to take capitalize on a juicy target? And if they were already in the system to take advantage of me, its not so much of a threat from pirates as it is from mind readers. Once my ships are in space is when word should get out. But before takeoff, business as usual happens on the planet and random criminals from who-knows-where shouldn't know a thing

I think it would make sense if the player's fleet attractiveness lag behind its actual ratio by just a day, since that's what separates two systems at the very least. This offers one safe jump/landing before the piracy threat takes effect. This can be one extra value in the savefile I think, though I don't know the best way this can be communicated to the player in game

Assuming it can be done, it can work in the opposite direction as well; having a poorly defended fleet then buying a bunch of warships to greet opportunistic pirates with bullets instead of the cargo they're here for means that we can be the ones to initiate combat instead of randomly stumbling across hostiles as rng dictates. I for would welcome the ability to give pirates a well deserved 500mg dose of ambushoxyzine

**Comment by OcelotWalrus** [comment]
> Hi, do you think that it could be more realistic if that pirate raids don't immediately become aware of the player's cargo to dps makeup? Information can't travel instantly across space. I know there's a reason for the pirate raid mechanic and I don't want to undermine freighter balancing. My concern is that the consequences kick in just a wee bit too early

I agree with you.

> Imagine I have a fully defended fleet one day but then decide to buy 10 bulk freighters while at the next shipyard. The moment I depart, pirates from kingdom come are waiting in orbit wanting a piece of me. The bottom line is how did they know? If they weren't already in the system before I bought the freighters then how are they here now to take capitalize on a juicy target? And if they were already in the system to take advantage of me, its not so much of a threat from pirates as it is from mind readers. Once my ships are in space is when word should get out. But before takeoff, business as usual happens on the planet and random criminals from who-knows-where shouldn't know a thing

Maybe some pirates in the spaceport inform the other pirates? But I agree with your point. Maybe pirate raids, instead of already being there when you depart, they'd come a little time after you depart, coming out of hyperspace.

> I think it would make sense if the player's fleet attractiveness lag behind its actual ratio by just a day, since that's what separates two systems at the very least. This offers one safe jump/landing before the piracy threat takes effect. This can be one extra value in the savefile I think, though I don't know the best way this can be communicated to the player in game
>
>Assuming it can be done, it can work in the opposite direction as well; having a poorly defended fleet then buying a bunch of warships to greet opportunistic pirates with bullets instead of the cargo they're here for means that we can be the ones to initiate combat instead of randomly stumbling across hostiles as rng dictates. I for would welcome the ability to give pirates a well deserved 500mg dose of ambushoxyzine

Seems interesting...

**Comment by Hurleveur** [comment]
I went back in time and coded this real quick
https://github.com/endless-sky/endless-sky/pull/9106
This only delays it when you become interesting enough for pirates tho (10 freighters prob would be enough for that)
